src/extension.ts

```diff
@@ -62,11 +62,12 @@ export class NewComment implements vscode.Comment {
         }
       }
     } catch (error) {
+      vscode.window.showWarningMessage(<string>error);

```

Comment:
@sharath-kannan What is this error and is it necessary to show this on the screen to the user?

Reply:
This is needed to show the correct error messages to the users and the developers of the extension.
This will also be helpful for the developer to debug if something has gone wrong.